src/community/infix-long-scale.ts

```diff
@@ -0,0 +1,31 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import Decimal from "break_infinity.js";
+import { toSubscript, abbreviate } from "../utils";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+export class InfixLongScaleNotation extends AbstractInfixNotation {
+  public get name(): string {
+    return "Infix long scale";
+  }
+  
+  protected groupDigits = 6;
+  protected canHandleZeroExponent = false;
+
+  protected formatMantissa(digit: number): string {
+    return toSubscript(digit);
+  }
+  protected formatExponent(exp: number): string {
+    if (exp < 0) {
+      return (exp / 6).toString();
+    }
+    return abbreviate(Math.floor(exp / 6));
+  }
+  
+  public formatDecimal(value: Decimal, places: number): string {
```

Comment:
Codacy found an issue: [Declaration of public instance method not allowed after declaration of protected instance method. Instead, this should come after protected instance fields.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)